src/tests/Common/XUnitLogChecker/XUnitLogChecker.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TestArchiveTestsDir>$(XUnitLogCheckerLibrariesOutputDirectory)</TestArchiveTestsDir>
```

Comment:
```suggestion
```

Reply:
Hmm interesting. I initially added it because it was a requirement in the conditions for the ZipTestArchive target and MakeDir and ZipDirectory tasks here:
https://github.com/dotnet/runtime/blob/11bdd24ed3f91d1bea118d872cc976a30294e269/eng/testing/tests.targets#L78-L90

But I guess that if we're manually adding our copy of that target to the csproj, then the value isn't required here anymore.

Reply:
With the below suggestion it isn't required anymore, yes. Because you don't import tests.targets in this projct anyway.

Reply:
Hmm something's weird. I think _it is_ used somewhere to ensure it gets zipped. Now it's not working anymore. I'm looking at the binlog.

Reply:
I was wrong, it does work. What does not work at this level is the IsXUnitLogCheckerSupported condition, it's false.

Reply:
At the csproj level, the property IsXUnitLogCheckerSupported is false because neither BuildTargetFramework nor RuntimeFlavor are available.

Reply:
I'm going to have to remove that from the IsXUnitLogCheckerSupported property and separately check for the framework version depending if it's the csproj or somewhere else.

Reply:
Yes, the BuildTargetFramework condition only really makes sense in sendtohelixhelp.proj. Avoid using it somewhere else.